boards/.gitignore

```diff
@@ -22,3 +22,4 @@ cscope.out
 /*.adb
 /*.lib
 /*.src
+/*/*/common/board
```

Comment:
Do you need to ignore this directory?

Reply:
Yes, it is a symbolic link generated by the build system